Content.Shared/_Goobstation/Devour/Systems/PreventSelfRevivalSystem.cs

```diff
@@ -0,0 +1,26 @@
+using Content.Shared._Goobstation.Devour;
+using Content.Shared._Goobstation.Devour.Events;
+using Content.Shared.Popups;
```

Comment:
Unused using directive
```suggestion
using Content.Shared._Goobstation.Devour.Events;
using Content.Shared.Popups;
```